lisa/sut_orchestrator/azure/common.py

```diff
@@ -192,3 +194,17 @@ def get_environment_context(environment: Environment) -> EnvironmentContext:
 def wait_operation(operation: Any) -> Any:
     # to support timeout in future
     return operation.wait()
+
+
+def create_storage_account_container(
```

Comment:
How about change it to `get_or_create_storage_container`?
1. If it exists, don't need create it. It avoids existing error on creating.
2. make the name shorter and accurate. It's not about storage account, it's about blob container. Is it right?

Reply:
@squirrelsc Using the` get_` prefix would ideally mean that we will also be returning the `container_client ` object from the function. I think we can make use of this object in the calling function for performing operations on the container. I am inclined to return the `container_client ` object from this function. Please let me know if you think otherwise.

Reply:
And yes it's about the blob container and not the storage account.

Reply:
@squirrelsc I have update the code. Please let me know if you find any issues.